infrastructure/server/flyway.tf

```diff
@@ -231,6 +231,8 @@ data "aws_lambda_invocation" "invoke_flyway_migration" {
           "api_db_password" : "md5${md5(join("", [local.flyway_db_creds.password, local.flyway_db_creds.username]))}",
           "auth_lambda_db_user" : "${local.flyway_db_auth_creds.username}",
           "auth_lambda_db_password" : "md5${md5(join("", [local.flyway_db_auth_creds.password, local.flyway_db_auth_creds.username]))}",
+          "admin_api_db_username" : "fam_proxy_admin_api",
+          "admin_api_db_password" : "ojkr97y663kvqcwlm8hyvf8dalxd",
```

Comment:
Both values are current intended for **temporary** only. I assume without giving a value the build will fail, so just gave them some value. The subsequent ticket when the user/password are setup on secret manager than these can be adjusted.

Reply:
In particular, everyone in the world knows this password now.

Reply:
The password is replaced with clear dummy value.